lms/djangoapps/grades/signals/handlers.py

```diff
@@ -122,6 +123,10 @@ def submissions_score_reset_handler(sender, **kwargs):  # pylint: disable=unused
 def disconnect_submissions_signal_receiver(signal):
     """
     Context manager to be used for temporarily disconnecting edx-submission's set or reset signal.
+
+    Clear Student State on ORA problems currently results in a set->reset signal pair getting fired
```

Comment:
Added comment to function that is causing circular dependency issue when trying to do a top level import of any functions from `api.py`. This comment is taken from the PR that introduced this dependency.

Reply:
Is this what was discussed in the huddle yesterday?